src/commands/scanner/run.ts

```diff
@@ -98,7 +98,7 @@ export default class Run extends ScannerCommand {
     const filters = this.buildRuleFilters();
     const target: string[]|string = this.flags.org || await this.unpackTargets();
     const format: OUTPUT_FORMAT = this.flags.format || this.deriveFormatFromOutfile();
-    const ruleManager = new RuleManager();
+    const ruleManager = await RuleManager.create({});
```

Comment:
Did you add a constructor that accepts `{}`? I don't see it anywhere.

Reply:
That's coming from parent class AsyncCreatable (https://forcedotcom.github.io/sfdx-dev-packages/kit/classes/asynccreatable.html) - another suggestion from Thomas to allow class properties to be async. Have a look at the changes in `PmdWrapper.java` - this has a more elaborate version of using `{}` to pass values to constructor to eventually assign to class properties.